examples/manual/longer.md

```diff
@@ -0,0 +1,53 @@
+# Setup
+1. Install tmt
+```bash
+sudo dnf install -y tmt-all
+```
+2. Create and switch to a brand new git repo
+```bash
+cd `mktemp -d`
+git init
+```
+# Test
+### Step
```

Comment:
I would propose to use commonly used phrases "Test Step" and "Expected Result".
I understand you are trying to make it compact, still...

Reply:
I understand @kkaarreell's point. The "commonly used" is a bit relative. I did a [brief investigation](https://github.com/psss/tmt/pull/198#issue-401379532) of the terms being already used in nitrate, polarion, xray and codebeamer. It's not very consistent (e.g. break down, tier down, post action), but the `expected result` is quite common, that's right.

I personally would keep it as concise as possible to prevent possible typos and repeating documentation checks when one forgets what's the right wording. My ideal would be consistent and short, everything a kind-of-a-verb: `setup`, `test`, `step`, `expect`, `cleanup`.

Or we could/should support both short and long version?

Reply:
> short and long version
It could be good compromise